lib/tokenVersions.js

```diff
@@ -111,12 +111,24 @@ export async function ensureTokenVersion({tokenizerId}) {
   return tokenVersion;
 }
 
+/**
+ * Gets a token version by either the version id or the tokenizerId.
+ *
+ * @param {object} options - Options to use.
+ * @param {number} [options.id] - An optional version id.
+ * @param {string} [options.tokenizerId] - An optional tokenizerId.
+ *
+ * @throws {Error} - If neither an id or tokenizerId is provided.
+ *
+ * @returns {object} A tokenizerVersion object.
+ */
 export async function get({id, tokenizerId}) {
   const query = {};
-  if(!(id && tokenizerId)) {
+  if(!((typeof id === 'number') || (typeof tokenizerId === 'string'))) {
     throw new TypeError('Either "id" or "tokenizerId" must be given.');
   }
-  if(id) {
+  // if the version id is 0 we need it in the query
+  if(id >= 0) {
     query['tokenVersion.id'] = id;
   }
```

Comment:
Actually, thinking about this more, this is still problematic without other validation, e.g., what if `id` is `true` and `tokenizerId` is given?

```suggestion
  if(id !== undefined) {
    if(!(Number.isInteger(id) && id >= 0)) {
      throw new TypeError('"id" must be a non-negative integer.');
    }
    query['tokenVersion.id'] = id;
  }
```